packages/wallet/src/W3mFrameHelpers.ts

```diff
@@ -73,5 +73,13 @@ export const W3mFrameHelpers = {
     return W3mFrameRpcConstants.SAFE_RPC_METHODS.includes(method)
   },
 
+  getPreferredAccountType(): 'eoa' | 'smartAccount' {
```

Comment:
Same for the previous tupple suggestion. If we keep the field name convention same with the keys, we could use it for the types as well:

```js
const AccountType = {
  smartAcconut: "smartAccount",
  eoa: "eoa"
}

type AccountType = keyof typeof AccountType
```